src/data/email/schemas.js

```diff
@@ -0,0 +1,7 @@
+import * as yup from 'yup';
+
+export const addEmailSchema = yup
+  .string()
+  .required('Introduce tu email')
+  .email('El e-mail introducido no es válido. Intenta de nuevo')
```

Comment:
@asermax A better text would be "Esto no parece ser un e-mail. Por favor, revisalo." - more human-friendly

Reply:
Lo agrego ahora! 💃